src/cursor/cur_backup_incr.c

```diff
@@ -206,6 +216,9 @@ __curbackup_incr_next(WT_CURSOR *cursor)
             WT_ERR(WT_NOTFOUND);
         WT_ASSERT(session, cb->granularity != 0);
         WT_ASSERT(session, total_len != 0);
+        __wt_verbose(session, WT_VERB_BACKUP,
+          "Set key WT_BACKUP_RANGE %s offset %" PRIu64 " gran %" PRIu64, cb->incr_file,
```

Comment:
I think here `gran` should say `length`. While those are almost always equal, they won't be if consolidation is turned on.